docs/content/users/install/ddev-installation.md

```diff
@@ -97,57 +97,78 @@ Installing and upgrading DDEV are nearly the same thing, because you're upgradin
 
-=== "Windows WSL2"
+=== "Windows"
 
-    ## Windows WSL2
+    ## Windows
 
-    Windows WSL2 is a fantastic way to run DDEV and your web components. It’s Linux, which means a different experience for many Windows users. It’s Ubuntu Linux by default as described here, so it’s worth taking a little time to explore how Ubuntu and Bash work, including standard system commands and installation and upgrade procedures.
+    You can install DDEV on Windows three ways:
 
-    **WSL2 is the recommended installation method for all Windows users**.
+    1. [Using WSL2 with Docker inside](#wsl2-docker-ce-inside-install-script)
+    2. [Using WSL2 with Docker Desktop](#wsl2-docker-desktop-install-script)
+    3. [Installing directly on traditional Windows](#traditional-windows) with an installer
 
-    **Using WSL2 to install and run DDEV is not the same as using Docker Desktop's WSL2 engine, which itself runs in WSL2, but can serve applications running in both traditional WIndows and inside WSL2.**
+    **We strongly recommend using WSL2.** While its Linux experience may be new for some Windows users, it’s worth the performance benefit and common experience of working with Ubuntu and Bash.
 
-    **All Windows 10/11 editions (including Windows 10 Home) support WSL2**. If you’re already familiar with DDEV on Windows, you might have been using NFS for better filesystem performance. **You won't need NFS anymore once you switch to WSL2**, since it provides awesome filesystem performance out of the box.
+
+    ### Important Considerations for WSL2 and DDEV
+
+    * WSL2 is supported on Windows 10 and 11.  
+      All Windows 10/11 editions, including Windows 10 Home support WSL2.
+    * WSL2 offers a faster, smoother experience.  
+      It’s vastly more performant, and you’re less likely to have obscure Windows problems.
+    * Projects should live in the Linux filesystem.  
+      WSL2’s Linux filesystem (e.g. `/home/<your_username>`) is much faster, so keep your projects there and **not** in the slower Windows filesystem (`/mnt/c`).
+    * Custom hostnames are managed via the Windows hosts file, not within WSL2.  
+      DDEV attempts to manage custom hostnames via the Windows-side hosts file, usually at `C:\Windows\system32\drivers\etc\hosts`. It can only do this if DDEV is installed and in the Windows-side `$PATH`. (DDEV inside WSL2 uses `ddev.exe` on the Windows side as a proxy to update the Windows hosts file.) If `ddev.exe --version` shows the same version as `ddev --version` you’re all set up. Otherwise, install DDEV on Windows using `choco upgrade -y ddev` or by downloading and running the Windows installer. (The WSL2 scripts below install DDEV on the Windows side, taking care of that for you.) If the Windows UAC escalation happens too often to you, you can calm it down by running `gsudo.exe cache on` and `gsudo.exe config CacheMode auto`, see [gsudo docs](https://github.com/gerardog/gsudo#credentials-cache).
```

Comment:
```suggestion
      DDEV attempts to manage custom hostnames via the Windows-side hosts file—usually at `C:\Windows\system32\drivers\etc\hosts`—and it can only do this if it’s installed on the Windows side. (DDEV inside WSL2 uses `ddev.exe` on the Windows side as a proxy to update the Windows hosts file.) If `ddev.exe --version` shows the same version as `ddev --version` you’re all set up. Otherwise, install DDEV on Windows using `choco upgrade -y ddev` or by downloading and running the Windows installer. (The WSL2 scripts below install DDEV on the Windows side, taking care of that for you.) If you frequently run into Windows UAC Escalation, you can calm it down by running `gsudo.exe cache on` and `gsudo.exe config CacheMode auto`, see [gsudo docs](https://github.com/gerardog/gsudo#credentials-cache).
```